vendor/golang.org/x/sys/execabs/execabs_go118.go

```diff
@@ -7,6 +7,12 @@
 
 package execabs
 
+import "os/exec"
+
 func isGo119ErrDot(err error) bool {
 	return false
 }
+
+func isGo119ErrFieldSet(cmd *exec.Cmd) bool {
+	return false
+}
```

Comment:
The code patch you provided seems trivial and has no functionality, as both the `isGo119ErrDot` and `isGo119ErrFieldSet` functions always return false. Thus, there are no bug risks or improvement suggestions at this point. However, it's worth noting that without seeing the context of the entire codebase, it's difficult to provide a complete review.